Gemfile

```diff
@@ -21,6 +21,7 @@ gem 'responders', '~> 2.0'
 # choose only one
 gem 'mysql2'
 # gem 'pg'
+# gem 'sqlite3'
```

Comment:
Why do you think we need to add this here?

Reply:
I thought that adding a comment help makes it obvious for the deployer.